pagopa/io-app
# chore: [IA-778] Move zendesk user identify after ask permission

## Short description
This PR moves the identification of the user after the user choose to give access to its information.
This is necessary because Zendesk creates a user in the Zendesk portal when user identification is performed.

Note that seems to be non-predictable, so sometimes you don't see the user in the Zendesk portal also if you perform the user identification.

## List of changes proposed in this pull request
- Moved the user identification in the `ZendeskAskPermissions` screen after the user press the continue button

## How to test
Go to the Zendesk portal, delete your user and try to open a request by pressing the "?"  button.
You shouldn't see your user again in the Portal.
Then try to go to the flow and share your personal information without sending any message.
You should see your user in the Portal.

Feel free to contact me if you don't have enough permission to test the PR
